common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -563,9 +675,10 @@ public String getID() {
     private void writeBuf(ByteBuffer buf) throws IOException {
       while (buf.hasRemaining()) {
         long updatedPos = partitionInfo.getDataFilePos() + length;
-        logger.debug("{} shuffleId {} reduceId {} current pos {} updated pos {}",
-          partitionInfo.appId, partitionInfo.shuffleId,
-          partitionInfo.reduceId, partitionInfo.getDataFilePos(), updatedPos);
+        logger.debug("{} shuffleId {} shuffleMergeId {} reduceId {} current pos"
+          + " {} updated pos {}", partitionInfo.appId, partitionInfo.shuffleId,
```

Comment:
Nit: Indent should be the same with the above "+". There are some other places down below for the long line "logger" that there is no need to put an additional indent, as the string and the later variables are at the same of parameters for the debug method. Please double check other ones.